components/batch-system/src/config.rs

```diff
@@ -16,10 +15,6 @@ pub struct Config {
     pub reschedule_duration: ReadableDuration,
     #[online_config(skip)]
     pub low_priority_pool_size: usize,
-    #[doc(hidden)]
-    #[serde(skip)]
-    #[online_config(skip)]
-    pub before_pause_wait: Option<Duration>,
```

Comment:
should we keep it for compatibility?

Reply:
No, it's marked as `serde(skip)` on purpose in the past to be able to be removed without breaking compatibility.